README.md

```diff
@@ -146,37 +146,44 @@ Edit the `.env` file with the necessary settings.
 #### `env.sample:`
 
 ```
-# Copy this file to .env and replace the credentials with 
-# your own before starting the app.
+# Replace the credentials here with your own.
```

Comment:
I think "Copy" is better.  Renaming the .env is a git change.  Copy is creating a .env that is in .gitignore